charts/catalog-api/templates/importer/job.yaml

```
@@ -57,6 +81,13 @@ spec:
       {{- if .Values.customCAs.bundle }}
         {{- include "catalog.custom.ca.jobs.volumes" . | nindent 8 }}
       {{- end }}
+      {{- if .Values.importer.postgres.tls.enabled }}
+      - name: {{ include "catalog.importer.name-psql-raw" . | quote }}
+        secret:
+          secretName: {{ include "catalog.importer.name-psql" . | quote }}
+      - name: {{ include "catalog.importer.name-psql" . | quote }}
+        emptyDir: {}
```

@DemonNSK 

В отступах ошибка, helm неможет спарсить. Относительно  `- name: {{ include "catalog.importer.name" . }}-pv`
```suggestion
        - name: {{ include "catalog.importer.name-psql-raw" . | quote }}
          secret:
            secretName: {{ include "catalog.importer.name-psql" . | quote }}
        - name: {{ include "catalog.importer.name-psql" . | quote }}
          emptyDir: {}
```